maraboupy/MarabouNetworkONNX.py

```diff
@@ -217,8 +224,15 @@ def makeGraphEquations(self, nodeName, makeEquations):
             raise RuntimeError(err_msg)
 
         # Compute node's shape and create Marabou equations as needed
+        if self.thresholdReached:
+            return
```

Comment:
Wouldn't doing the thresholding stuff on the C++ side with the final set of equations _after_ the parsing of the network be a much better way of doing it? Then you would get this feature for all parsers and both Python and C++, rather than simply the ONNX network for the Python backend?

Reply:
This is out of scope of this PR. Maybe it will be supported in the future PR.